src/components/templates/newsAndEvents/newsAndEvents.scss

```
@@ -0,0 +1,48 @@
+@use "../../variables" as *;
+
+.newsEvents {
+  padding: 0rem 1rem;
+  background-color: $white_smoke;
+  .news-events {
+    padding: 56px 0px;
+    display: flex;
+    justify-content: space-between;
+    @media screen and (max-width: $breakpoint-lg) {
+      justify-content: center;
+      align-items: center;
+      flex-flow: column;
+      gap: 35px;
+    }
+    .news {
+      width: 48%;
+      @media screen and (max-width: $breakpoint-lg) {
+        width: 88%;
+      }
+    }
+    .events {
+      width: 48%;
+      @media screen and (max-width: $breakpoint-lg) {
+        width: 88%;
```

```suggestion
      flex: 1;
    }
    .events {
      flex: 1;
      @media screen and (max-width: $breakpoint-lg) {
```

現状でも見た目は問題ないので修正不要ですが、flexboxを使って要素を均等に並べるのであれば、それぞれに `flex: 1;` を設定し、余白は `gap` プロパティで管理するというのがスッキリかけておすすめです。

参考：https://www.notion.so/shinonome-inc/Flexbox-1-e28e57a82d004507bfef1179b8fd6845?pvs=4

---

flex=1を試したところ、画面幅medium以外のときに親要素からはみ出してしまい調整の方法が分からなかったので、そのままにしました。

---

なるほど！
どこかでwidthを固定値に設定していたり、 `flex` を指定している要素がフレックスアイテムではないなどの原因が考えられそうですね :thinki

---

今回幅を調整した箇所を無効化させてみたのですが上手くいきませんでした。恐らく以前作成した要素に問題があるようなので今回はflexでない方法にし今後の作成の際にflexを使いたいと思います